deliver/lib/deliver/submit_for_review.rb

```diff
@@ -63,6 +63,17 @@ def update_export_compliance(options, app, build)
 
       UI.verbose("Updating build for export compliance status of '#{uses_encryption}'")
       if build.uses_non_exempt_encryption.nil?
+        if uses_encryption.to_s.empty?
+          message = [
+            "Export compliance is required to submit",
+            "Add information to the :submission_information option...",
+            "  Docs: http://docs.fastlane.tools/actions/deliver/#compliance-and-idfa-settings",
+            "  Example: submission_information: { export_compliance_uses_encryption: false }",
+            "This can also be set in your Info.plist with key 'ITSAppUsesNonExemptEncryption'"
+          ].join("\n")
+          UI.user_error!(message)
+        end
+
         build = build.update(attributes: {
           usesNonExemptEncryption: uses_encryption
```

Comment:
@joshdholtz should we use `export_compliance_is_exempt` instead of `export_compliance_uses_encryption` here?

Reply:
We use snake_case representations of the keys from the API

Reply:
@minuscorp of course, but that was not my question :) If you look at the line 62, `uses_encryption` is extracted from `submission_information[:export_compliance_uses_encryption]`. and later used to set `usesNonExemptEncryption`. 

It looks to me that we should use `submission_information[:export_compliance_is_exempt]` to set `usesNonExemptEncryption`. Actually the negation of that :D

Reply:
@mjovkovic Thanks for this review! This is totally the case. You are right 😊 I got another PR that I was doing something different and noticed this and fixed it in there 🤷‍♂️ I'm going to merge his but I will have this fixed in in the `2.150.0` branch shortly. Sorry about that! ❤️

Reply:
Actually... I'm just gunna push it to this PR

Reply:
just was going to comment, was the negation, 🤣

Reply:
@mjovkovic The different between `export_compliance_uses_encryption` and `export_compliance_is_exempt` is really confusing 🤔  I think `export_compliance_uses_encryption` is the correct one 😬 But this (along with other things) are named differently now that make it confusing...

I'm actually thinking of deprecating `submission_information` and warning users to change to new `deliver` options called something like `idfa` (a hash), `export_compliance` (a hash), and `content_rights`. They are there own things now and I think it makes sense to bring the options and new naming together.

Reply:
No way to give access to different objects doesn't it? Something they can import and use in the submission, critical point in the process. Thoughts?

Reply:
It will still behave the same way as `submission_information` does now but...
1) be split up into 3 different options instead of 1 large confusing hash (will be easier to document)
2) be renamed correctly to how App Store Connect represents them now (which shouldn't change for a while)

I don't want to be stuck with the legacy iTunesConnect style naming for forever 😊

Reply:
@joshdholtz sounds good! 🚀

Reply:
> @mjovkovic The different between `export_compliance_uses_encryption` and `export_compliance_is_exempt` is really confusing 🤔 I think `export_compliance_uses_encryption` is the correct one 😬 But this (along with other things) are named differently now that make it confusing...

I'm trying to compare those fields to the ones on Appstoreconnect when you need to provide compliance. Here are the questions. Depending on the answers wizard can end earlier. E.G. selecting `No` on the first one doesn't ask for exempt end the other ones.

![Screenshot 2020-06-23 at 17 11 23](https://user-images.githubusercontent.com/15940298/85421642-f5080a00-b574-11ea-9594-9136a3664c1f.png)
^^^ That one is probably related to some general encryption usage flag.

![Screenshot 2020-06-23 at 17 11 35](https://user-images.githubusercontent.com/15940298/85421659-fb968180-b574-11ea-98a8-14c183f22f19.png)
^^^ This one looks like `usesNonExemptEncryption`, only its negated 😕 

The rest of them looks like they are directly related to `export_compliance_contains_proprietary_cryptography`, `export_compliance_contains_third_party_cryptography` and `export_compliance_available_on_french_store`
![Screenshot 2020-06-23 at 17 11 52](https://user-images.githubusercontent.com/15940298/85421682-018c6280-b575-11ea-9b71-7dd1d885a477.png)

![Screenshot 2020-06-23 at 17 12 01](https://user-images.githubusercontent.com/15940298/85421726-0b15ca80-b575-11ea-8f85-d1030b7b1cf0.png)

![Screenshot 2020-06-23 at 17 12 25](https://user-images.githubusercontent.com/15940298/85421736-0f41e800-b575-11ea-99cf-8d1f261b3088.png)

Reply:
@mjovkovic Yeah, things really look changed up. I think deprecating and using a new thing might just be best 😬 I was struggling to map everything.

Do you have an app that uses encryption? I don't have one so its really hard for me to test 😔

Reply:
@joshdholtz yes, I do :) It uses encryption, it's not exempt from export compliance, doesn't use any proprietary algorithms, it uses standard ones and it not available in French store. I can test when it's ready :)

Reply:
@mjovkovic That would be amazing! I've merged a bunch of work into the `2.150.0` branch if you are able to test that 👇 I'm planning on getting a bunch of new things out in a `2.150.0.rc2` either later tonight or early tomorrow 🤞 

```ruby
gem "fastlane", :git => "https://github.com/fastlane/fastlane.git", :branch => "2.150.0"
```

Reply:
@joshdholtz I've tested again wih branch 2.150.0 in verbose mode. It failed again :(

```
INFO [2020-06-24 04:58:35.66]: Successfully selected build
DEBUG [2020-06-24 04:58:35.66]: Updating build for export compliance status of 'true'
DEBUG [2020-06-24 04:58:35.88]: Successfully updated build for export compliance status of '' on App Store Connect
DEBUG [2020-06-24 04:58:38.14]: Updating app store version for IDFA status of 'false'
DEBUG [2020-06-24 04:58:38.36]: Updated app store version for IDFA status of 'false'
INFO [2020-06-24 04:58:38.36]: Successfully updated IDFA delcarations on App Store Connect
DEBUG [2020-06-24 04:58:38.36]: Updating contents rights declaration on App Store Connect
INFO [2020-06-24 04:58:39.02]: Successfully updated contents rights declaration on App Store Connect
WARN [2020-06-24 04:58:39.87]: Lane Context:
INFO [2020-06-24 04:58:39.87]: {:DEFAULT_PLATFORM=>:ios, :PLATFORM_NAME=>:ios, :LANE_NAME=>"ios deploy", :VERSION_NUMBER=>"300000.2.30", :BUILD_NUMBER=>"300000230", :PRODUCE_APPLE_ID=>"1195204795", :CERT_FILE_PATH=>"******", :CERT_CERTIFICATE_ID=>"3Q2MB7V568", :SIGH_PROFILE_PATH=>"********", :SIGH_PROFILE_PATHS=>["********"], :SIGH_UDID=>"********", :SIGH_UUID=>"***********", :SIGH_NAME=>"*********", :SIGH_PROFILE_TYPE=>"app-store", :IPA_OUTPUT_PATH=>"***********", :XCODEBUILD_ARCHIVE=>"*************", :DSYM_OUTPUT_PATH=>"***********"}
ERROR [2020-06-24 04:58:39.87]: The request cannot be fulfilled because of the state of another resource. - Submit for review errors found.
The provided entity is missing a required attribute - You must provide a value for the attribute 'usesNonExemptEncryption' with this request
```

There's still a discrepancy with what's being set for `usesNonExemptEncryption` and what is read afterwards from `build.uses_non_exempt_encryption`. That looks to be the source of the problem. 

```
DEBUG [2020-06-24 04:58:35.66]: Updating build for export compliance status of 'true'
DEBUG [2020-06-24 04:58:35.88]: Successfully updated build for export compliance status of '' on App Store Connect

```